openshift/flower/s2i/bin/assemble

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+function is_django_installed() {
```

Comment:
Seems like this isn't necessary. Do we need all the django related functions?

Reply:
This is the generic upstream scripting. I've left it unmodified.

Technically, no we don't need it. But, it's also easier to let it match the upstream script than to spend time stripping out the unused functionality.